examples/convert.rs

```diff
@@ -53,7 +53,12 @@ fn main() {
     let param_path = std::path::PathBuf::from(&args[1]).with_extension("param.ron");
     let params = match fs::read_to_string(param_path) {
         Ok(string) => ron::de::from_str(&string).unwrap(),
-        Err(_) => Parameters::default(),
+        Err(_) => {
+            let mut param = Parameters::default();
+            // very useful to have this by default
+            param.spv_capabilities.insert(spirv::Capability::Shader);
```

Comment:
Very useful indeed! Maybe something for later to add for all examples by default when a `param.ron` is not found?

Reply:
yes, I was thinking about this as well, just having a default one in a file. Good for a follow-up!